docs/setup_ovos_services.md

```diff
@@ -0,0 +1,191 @@
+# Running & Setting Up OVOS Services
+
+OpenVoiceOS is a software stack that includes seven important services, with some being optional, depending on the platform or environment. If you are a developer or would like to quickly test OpenVoiceOS for debugging purposes, you can run each service separately in a terminal. However, for users, distributions, and packagers, it is recommended to use systemd or any other init service that their userland supports for launching and managing OpenVoiceOS services.
+
+### Here are the commands for launching each of the services from the CLI:
+
+ - Launching the Messagebus service (required): python3 -m mycroft.messagebus.service
+ - Launching the Skills service (required): python3 -m mycroft.skills
+ - Launching the PHAL service (required): python3 -m ovos_PHAL
+ - Launching the Audio service (required): python3 -m mycroft.audio
+ - Launching the Voice service (required): python3 -m mycroft.client.speech
+ - Launching the GUI service (optional): python3 -m mycroft.gui
+ - Launching the CLI service (optional): python3 -m mycroft.client.text
```

Comment:
cli should never be a service, its just a standalone app, I would not even mention it here

also it should now be ovos-cli-client, mycroft.client.text is just for compat